config/default/jenkins-infra.yaml

```diff
@@ -149,104 +150,48 @@ jenkins:
-                      repository('docker-mirrorbits')
-                    }
-                  }
-                  factory {
-                    workflowBranchProjectFactory {
-                      scriptPath('Jenkinsfile_k8s')
-                    }
-                  }
-                }
-            - script: >
-                multibranchPipelineJob('docker-datadog') {
-                  displayName "DataDog Docker Build"
-                  branchSources {
-                    github {
-                      id('2020111801')
-                      scanCredentialsId('github-access-token')
-                      repoOwner('jenkins-infra')
-                      repository('docker-datadog')
-                    }
-                  }
-                  factory {
-                    workflowBranchProjectFactory {
-                      scriptPath('Jenkinsfile_k8s')
-                    }
-                  }
-                }
-            - script: >
-                multibranchPipelineJob('docker-terraform') {
-                  displayName "Terraform Docker Build"
-                  branchSources {
-                    github {
-                      id('2020121001')
-                      scanCredentialsId('github-access-token')
-                      repoOwner('jenkins-infra')
-                      repository('docker-terraform')
-                    }
-                  }
-                  factory {
-                    workflowBranchProjectFactory {
-                      scriptPath('Jenkinsfile_k8s')
-                    }
-                  }
-                }
-            - script: >
-                multibranchPipelineJob('docker-builder') {
-                  displayName "Docker Builder's Docker Build"
-                  branchSources {
-                    github {
-                      id('2020122201')
-                      scanCredentialsId('github-access-token')
-                      repoOwner('jenkins-infra')
-                      repository('docker-builder')
-                    }
-                  }
-                  factory {
-                    workflowBranchProjectFactory {
-                      scriptPath('Jenkinsfile_k8s')
+                [
+                  [repo: 'docker-packer', owner: 'jenkins-infra', name: 'Packer Docker Build', trust: 'TrustPermission'],
+                  [repo: 'docker-mirrorbits', owner: 'jenkins-infra', name: 'Mirrorbits Docker Build', trust: 'TrustPermission'],
+                  [repo: 'docker-datadog', owner: 'jenkins-infra', name: 'Datadog Docker Build', trust: 'TrustPermission'],
+                  [repo: 'docker-terraform', owner: 'jenkins-infra', name: 'Terraform Docker Build', trust: 'TrustPermission'],
+                  [repo: 'docker-builder', owner: 'jenkins-infra', name: "Docker Builder's Docker Build", trust: 'TrustPermission'],
+                  [repo: 'aws', owner: 'jenkins-infra', name: 'AWS Infra'],
+                ].each{ config ->
```

Comment:
would it be better to just configure a github organisation job? so we don't have to update this every time a new repo comes along?